core/amber/src/main/protobuf/edu/uci/ics/texera/workflowruntimestate.proto

```diff
@@ -66,4 +66,5 @@ message JobStatsStore {
 message JobStateStore{
   WorkflowAggregatedState state = 1;
```

Comment:
I just remembered this, be careful with those, you may want to add `[(scalapb.field).no_box = true]` to disable it to be wrapped by an `Option`.

Reply:
I set `no_default_values_in_constructor` to false so scalpb won't wrap it by an `Option`. And the default value is `UNINITIALIZED`.

Reply:
I am not sure if `no_default_values_in_constructor` would do the same for `no_box` configuration. can you verify generated message do not have `Option` wrapper for fields?

Reply:
I'm using the generated proto objects in my code, there is no `Option` wrapper.